echopype/tests/convert/test_parsed_to_zarr.py

```diff
@@ -15,6 +15,8 @@
 from echopype.echodata.convention import sonarnetcdf_1
 from echopype.convert.api import _check_file, SONAR_MODELS
 
+pytestmark = pytest.mark.skip(reason="Removed Parsed2Zarr")
```

Comment:
Is this relevant to this PR?

Reply:
> Is this relevant to this PR?

Yepp, without the skip, it'll have a botocore error because it's spinning up moto server multiple times... I have to figure that out in PR https://github.com/OSOceanAcoustics/echopype/pull/1185. For now, it's best to skip this test since lots of changes a coming!

Reply:
Got it